extensions/amp-analytics/0.1/vendors.js

```diff
@@ -922,6 +922,75 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
     },
   },
 
+  'mediator': {
+    'requests': {
+      'host': '//collector.mediator.media/amp/?',
+      'prefix': '${host}cid=${mediator_id}&url=${canonicalUrl}' +
+                '&ref=${documentReferrer}&p=4&',
+      'suffix': 'vh=${viewportHeight}&sh=${scrollHeight}&st=${scrollTop}',
+      'pageview': '${prefix}e=v',
+      'timer': '${prefix}e=t&${suffix}',
+      's0': '${prefix}e=s0',
+      's1': '${prefix}e=s1',
+      's2': '${prefix}e=s2',
+      's3': '${prefix}e=s3',
+    },
+    'vars': {
+      'mediator_id': '',
+    },
+    'triggers': {
+      'trackPageview': {
+        'on': 'visible',
+        'request': 'pageview',
+      },
+      'scrollPing0': {
+        'on': 'scroll',
+        'scrollSpec': {
+          'verticalBoundaries': [
+            1,
+          ],
+        },
+        'request': 's0',
```

Comment:
Note you can use `${verticalScrollBoundary}` to get the vertical scroll value instead of having four different requests. Does that work for you?

Reply:
No. On the server, we process data about the current position of the user on the page, collecting information on certain sections of this page.

Reply:
I wonder if there's is way to replace variable with a function;
`vars: {'scroll_id': Math.round(${verticalScrollBoundary}/35)}` for example. I couldn't find such url replacement function. cc @avimehta ?

Reply:
I don't quite understand - what I still have to do?

Reply:
@zhouyx There is no way to do this right now. Lets try and get something like this working.

@mktrue are the values `s0`, `s1` and so on not changeable? If they are changeable, here is what you could do:

```
  'mediator': {
    'requests': {
      'host': '//collector.mediator.media/amp/?',
      'prefix': '${host}cid=${mediator_id}&url=${canonicalUrl}' +
                '&ref=${documentReferrer}&p=4&',
      'suffix': 'vh=${viewportHeight}&sh=${scrollHeight}&st=${scrollTop}',
      'pageview': '${prefix}e=v',
      'timer': '${prefix}e=t&${suffix}',
    },
    'vars': {
      'mediator_id': '',
    },
    'triggers': {
      'trackPageview': {
        'on': 'visible',
        'request': 'pageview',
      },
      'scrollPings': {
        'on': 'scroll',
        'scrollSpec': {
          'verticalBoundaries': [
            5, 35, 65, 95,
          ],
        },
        'request': 'prefix',
        'extraUrlParams': {
          'e': 's${verticalScrollBoundary}',
        },
      },
      'pageTimer': {
        'on': 'timer',
        'timerSpec': {
          'immediate': false,
          'interval': 5,
          'maxTimerLength': 600,
        },
        'request': 'timer',
      },
    },
  },
```

You can test the change locally and see if the new request format works for you. If it does, then your configuration becomes much simpler. If it does not, you might have to stick to the configuration you have added for now.

Reply:
We have a set of exact keys (s0, s1, s2, s3) and we can't change them. Only this variant is suitable for us:
```
“extraUrlParams”: {
    “e”: “s${Math.round(verticalScrollBoundary)/35}”
}
```

Reply:
@zhouyx I can't open https://percy.io/ampproject/amphtml/builds/356562 because no rights to view. How can I get a invite ?

Reply:
@avimehta Thanks for the info. I also asked them if they can do `${verticalScrollBoundary}` before, but they would need some kind of calculation there instead of just getting the absolute value. 
Another thing is the config can use vars in trigger to save few requests. But it would still require multiple triggers for every scroll threshold. Since we can't support `vars: {'scroll_id': Math.round(${verticalScrollBoundary}/35)}` now, I'll go ahead and approve this Pull Request.